easybuild/easyconfigs/s/spglib/spglib-1.9.4.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'spglib'
```

Comment:
@migueldiascosta `pypi.io` is still pre-production, so considering that is too early imho; `files.pythonhosted.org` may work, but since `pypi.python.org` still works fine (well, after we fixed https://github.com/hpcugent/easybuild-framework/pull/1749), I see no reason to switch already

hardcoding `spglib` in `source_urls` and the `modulename` used for the sanity check is the right approach, there's no other way to derive it from `name` or something

Support for removing files with `--update-pr` was implemented in https://github.com/hpcugent/easybuild-framework/pull/1853, and will be included in the upcoming EasyBuild v2.9.0; you'll need to do something like `eb --update-pr 3365 :foo.eb`, where the `:` indicates that the file is to be removed.